cli/lib/exec/spawn.js

```diff
@@ -168,11 +182,17 @@ module.exports = {
         debug('DISPLAY is %s', process.env.DISPLAY)
       }
 
+      const electronStarted = Number(new Date())
+
       return spawn()
       .then((code) => {
-        const POTENTIAL_DISPLAY_PROBLEM_EXIT_CODE = 234
+        const electronFinished = Number(new Date())
+        const elapsed = electronFinished - electronStarted
+
+        debug('electron open returned %d after %d ms', code, elapsed)
 
-        if (shouldRetryOnDisplayProblem && code === POTENTIAL_DISPLAY_PROBLEM_EXIT_CODE) {
+        if (shouldRetryOnDisplayProblem &&
+          isPotentialDisplayProblem(os.platform(), process.env.DISPLAY, code, elapsed)) {
```

Comment:
now this should work with both debug version of Electron in the monorepo and with compiled Cypress binary app